src/include/btmem.h

```diff
@@ -72,15 +72,14 @@ struct __wt_page_header {
 /*
  * No automatic generation: flag values cannot change, they're written to disk.
  */
-#define WT_PAGE_COMPRESSED 0x01u      /* Page is compressed on disk */
-#define WT_PAGE_EMPTY_V_ALL 0x02u     /* Page has all zero-length values */
-#define WT_PAGE_EMPTY_V_NONE 0x04u    /* Page has no zero-length values */
-#define WT_PAGE_ENCRYPTED 0x08u       /* Page is encrypted on disk */
-#define WT_PAGE_UNUSED 0x10u          /* Historic lookaside store page updates, no longer used */
-#define WT_PAGE_FT_UPDATE 0x20u       /* Page contains updated fast-truncate information */
-#define WT_PAGE_STAT_BYTE_COUNT 0x40u /* Page contains byte count information */
-#define WT_PAGE_STAT_ROW_COUNT 0x80u  /* Page contains row count information */
-    uint8_t flags;                    /* 25: flags */
+#define WT_PAGE_COMPRESSED 0x01u   /* Page is compressed on disk */
+#define WT_PAGE_EMPTY_V_ALL 0x02u  /* Page has all zero-length values */
+#define WT_PAGE_EMPTY_V_NONE 0x04u /* Page has no zero-length values */
+#define WT_PAGE_ENCRYPTED 0x08u    /* Page is encrypted on disk */
+#define WT_PAGE_UNUSED 0x10u       /* Historic lookaside store page updates, no longer used */
+#define WT_PAGE_FT_UPDATE 0x20u    /* Page contains updated fast-truncate information */
+#define WT_PAGE_SUPPORT_STAT 0x40u /* Page supports page stat */
+    uint8_t flags;                 /* 25: flags */
 
     /* A byte of padding, positioned to be added to the flags. */
     uint8_t unused; /* 26: unused padding */
```

Comment:
Another choice is to use the version value instead of flags for page with stat information. @agorrod

Reply:
Or should we set the flag but also bump the version since now we have page deleted structure and page stat?

Reply:
I'm a bit confused why we need this version field. Seems it doesn't serve any functional purpose just a symbol of version.